test/src/test/java/test/org/dst/core/operator/KVSListTest.java

```diff
@@ -0,0 +1,90 @@
+package test.org.dst.core.operator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.dst.core.KVStoreImpl;
+import org.dst.core.KVStore;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class KVSListTest {
+
+    @Test
+    public void testPutAndGet() {
+        KVStore ks = new KVStoreImpl();
+        List<String> list = new ArrayList<>();
+        list.add("v1");
+        list.add("v2");
+        list.add("v3");
```

Comment:
```java
ImmutableList.of("v1", "v2", "v3");
```

Reply:
Is that ok?   
 Arrays.asList("a","b")

Reply:
Why not use `ImmutableList`. It's the best practice to construct a list I think.

Reply:
ImmutableList is Immutable Collection，
It is not suitable for the change the collection test